### PR TITLE
[CHI-2024] Initial setup for 2024 Chicago event

### PR DESCRIPTION
**Pull request by mattstratton**
Email has been sent to info@devopsdays.org (sent from Dex Horthy) with updated organizer list

### CODE REVIEW

**Comment by mattstratton** [comment]
email alias has been updated!

File: static/events/2024-chicago/organizers/corey-ward.jpg
Comment (nimbinatus) [inline static/events/2024-chicago/organizers/corey-ward.jpg:1]: I think this picture is bigger than the 600px square guideline, but I'm not sure how picky y'all are considering filesize-wize this one is smaller than some of the others that are definitely below 600px...

File: content/events/2024-chicago/propose.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- _no vendor pitches_: As much as we value vendors and sponsors, we are not going to accept a talk that appears to be a pitch for your product.
+
+<hr>
+
+<strong>How to submit a proposal:</strong> Send an email to [{{< email_organizers >}}] with the following information
+<ol>
+	<li>Type (presentation, panel discussion, ignite)</li>
+	<li>Proposal Title (can be changed later)</li>
+	<li>Description (several sentences explaining what attendees will learn)</li>
+</ol>
```
[the remark this answers is not in the document]
Comment (mattstratton) [reply content/events/2024-chicago/propose.md:35]: yeah I should probably remove them and we'll add them when we need them

File: content/events/2024-chicago/sponsor.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Description = "Sponsor devopsdays Chicago 2024"
++++
+
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+
+<hr>
+
+devopsdays is a self-organizing conference for practitioners that depends on sponsorships. We do not have vendor booths, sell product presentations, or distribute attendee contact lists. Sponsors have the opportunity to have short elevator pitches during the program and will get recognition on the website and social media before, during and after the event. Sponsors are encouraged to represent themselves by actively participating and engaging with the attendees as peers. Any attendee also has the opportunity to demo products/projects as part of an open space session.
+<p>
+Gold sponsors get a full table and Silver sponsors a shared table where they can interact with those interested to come visit during breaks. All attendees are welcome to propose any subject they want during the open spaces, but this is a community-focused conference, so heavy marketing will probably work against you when trying to make a good impression on the attendees.
```
[the remark this answers is not in the document]
Comment (mattstratton) [reply content/events/2024-chicago/sponsor.md]: yeah, removing this file in https://github.com/devopsdays/devopsdays-web/pull/13523/commits/32400638ba5ef1b0aedaaefd54d6753d7e8a2713

File: content/events/2024-chicago/propose.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattstratton) [reply content/events/2024-chicago/propose.md]: and removed! https://github.com/devopsdays/devopsdays-web/pull/13523/commits/32400638ba5ef1b0aedaaefd54d6753d7e8a2713

File: static/events/2024-chicago/organizers/corey-ward.jpg
Comment (mattstratton) [reply static/events/2024-chicago/organizers/corey-ward.jpg:1]: I've decided that this is Probably Fine as it's the same one we used last year and nobody noticed :D

Comment (nimbinatus) [reply static/events/2024-chicago/organizers/corey-ward.jpg:1]: No worries, then!

**Pull request merged**